thirdeye/thirdeye-frontend/app/pods/manage/alerts/controller.js

```diff
@@ -44,11 +65,17 @@ export default Ember.Controller.extend({
     // Handles alert selection from type ahead
     onAlertChange(alerts) {
       this.get('selectedAlerts').pushObject(alerts);
+      this.set('resultsActive', true);
     },
 
     // Handles UI mode change
     onModeChange(mode) {
       this.set('selectedSearchMode', mode);
     },
+
+    removeAll() {
+      $('.te-search-results').remove();
```

Comment:
We need to clear previously rendered results. The conditional wrapper would just not display them.